src/screens/generation/generation.vue

```diff
@@ -5,18 +5,15 @@
     </h3>
     <div class="generation-header">
       <div class="basic-gen-info grid">
-        <div>
-          <div class="field-name">Height</div>
-          <div class="number">{{ generation.keyBlock.height }}</div>
-        </div>
-        <div>
-          <div class="field-name">Micro Blocks</div>
-          <div class="number">{{ generation.micros.length }}</div>
-        </div>
-        <div>
-          <div class="field-name">Transactions</div>
-          <div class="number">{{ generation.transactionNumber }}</div>
-        </div>
+        <field name="Height">
+            <div class="number">{{ generation.keyBlock.height }}</div>
```

Comment:
Usually, we are using two space as indents.

Reply:
Thanks, I will use that convention from now on.